examples/cookbook/design/drawer/lib/drawer.dart

```diff
@@ -3,6 +3,9 @@ import 'package:flutter/material.dart';
 void drawerStart() {
   // #docregion DrawerStart
   Scaffold(
+    appBar: AppBar(
+      title: const Text('AppBar with hamburger button'),
```

Comment:
I think the title text in this version should be 'AppBar **without** hamburger button'

This title will be misleading, there is no drawer, so there will be no hamburger button in this version of the code.

Reply:
Well, you are right. There is no drawer when drawer is not set.